add-on/manifest.json

```diff
@@ -38,6 +38,22 @@
         "page": "dist/background/background.html"
     },
 
+    "content_scripts": [
+      {
+        "matches": [
+          "file://*/*",
+          "http://*/*",
```

Comment:
A single entry with `<all_urls>` would be better here:

> A match pattern is essentially a URL that begins with a permitted scheme (http, https, file, or ftp, and that can contain '*' characters. The special pattern `<all_urls>` matches any URL that starts with a permitted scheme. ([src](https://developer.chrome.com/extensions/match_patterns))

Reply:
We'll eventually inject programmatically due to the need to turn this feature off, but yes that would be better.